doc/classes/Array.xml

```diff
@@ -183,36 +174,46 @@
 			<return type="void" />
 			<param index="0" name="value" type="Variant" />
 			<description>
-				Appends an element at the end of the array (alias of [method push_back]).
+				Appends [param value] at the end of the array (alias of [method push_back]).
 			</description>
 		</method>
 		<method name="append_array">
 			<return type="void" />
 			<param index="0" name="array" type="Array" />
 			<description>
-				Appends another array at the end of this array.
+				Appends another [param array] at the end of this array.
 				[codeblock]
-				var array1 = [1, 2, 3]
-				var array2 = [4, 5, 6]
-				array1.append_array(array2)
-				print(array1) # Prints [1, 2, 3, 4, 5, 6].
+				var numbers = [1, 2, 3]
+				var extra = [4, 5, 6]
+				numbers.append_array(extra)
+				print(nums) # Prints [1, 2, 3, 4, 5, 6]
 				[/codeblock]
 			</description>
 		</method>
 		<method name="back" qualifiers="const">
 			<return type="Variant" />
 			<description>
-				Returns the last element of the array. Prints an error and returns [code]null[/code] if the array is empty.
-				[b]Note:[/b] Calling this function is not the same as writing [code]array[-1][/code]. If the array is empty, accessing by index will pause project execution when running from the editor.
+				?? Returns the last element of the array. Generates an error and returns [code]null[/code] if the array is empty.
+				?? Returns the last element of the array. If the array is empty, generates an error and returns [code]null[/code], without stopping the project execution (unlike with the [code][][/code] operator).
+				?? [b]Note:[/b] This method is not the same as [code]array[-1][/code].
```

Comment:
Unsure of how to say this, here. I do not want to lose  _"Calling this function is not the same as writing [code]array[-1][/code]"_, but it also feels like repetitive information at the same time.